lib/devise.rb

```diff
@@ -153,7 +153,11 @@ module Test
   mattr_accessor :pepper
   @@pepper = nil
 
-  # Used to enable sending notification to user when their password is changed
+  # Used to send notification to the original user email when their email is changed.
+  mattr_accessor :send_email_change_notification
```

Comment:
I wonder if using `email_changed` is more fitting than `email_change` to display that the even has already happened (even considering that we already use similar names around the codebase)